test/release/examples/README

```diff
@@ -63,11 +63,14 @@ subdirectories by simply using:
 Other files
 -----------
 
-./
-  Makefile        : Makefile for compiling the examples
+./                : This directory
```

Comment:
I tried running start_test on various targets, and ./Logs is always created in the top directory (examples)- even if you are running programs located in subdirs.  

Note: I did not try starting start_test from a subdirectory- even if it would work, it seems beyond the scope of this README, which only talks about running start_test from the top directory.